.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Ask questions on Discord
+    url: https://discord.gg/9vrAZ8xGyh
```

Comment:
I wasn't sure if this was a persistent link or not, but since you have it on the website it seemed like a safe bet.

Reply:
It is; we're good